dependency_management.md

```diff
@@ -0,0 +1,20 @@
+Dependency Management
+===========
+
+The ResearchKit framework can also be added to your app using CocoaPods or Carthage.
+
+Installation with CocoaPods
+------------
+Add the following line to your [Podfile](http://guides.cocoapods.org/syntax/podfile.html) and run `pod install`:
+
+```ruby
+pod 'ResearchKit', '~> 1.0'
```

Comment:
Does having `1.0` instead of `1.0.0` be an issue? Or it should just work?

The current tag is `1.0.0`.